src/Input/AbstractParamAwareInput.php

```diff
@@ -306,4 +306,9 @@ static function (mixed $value) use ($originalValidator) {
 
         return $originalValidator;
     }
+
+    public function __toString(): string
+    {
+        return '';
+    }
```

Comment:
Currently got no idea what `__toString` here should actually return, or what its purpose is. Please feel free to jump in and add to this patch directly 👍

Reply:
Here are two examples from the console component itself:

- https://github.com/symfony/console/blob/64e06788686633deb8d9a7c75ab31bcf4b233a26/Input/ArrayInput.php#L92-L114
- https://github.com/symfony/console/blob/64e06788686633deb8d9a7c75ab31bcf4b233a26/Input/ArgvInput.php#L345-L363

No idea if it really helps.